### PR TITLE
Use libzim 7.2.2

### PR DESCRIPTION
**Pull request by rgaudin**
- Use libzim 7.2.1 for release and ci workflows
- Updated tests to new libzim 7.2.1 behavior (exceptions on duplicates)
- Use latest Cython version

### CODE REVIEW

**Comment by kelson42** [comment]
We should probably wait that libzim 7.2.1 build for macOS is done before merging this.

**Comment by rgaudin** [comment]
The build is already online but we should wait for the PR to be merged in kiwis-build before merging this ; for consistency

**Comment by kelson42** [comment]
@rgaudin Does it fix #141

**Comment by rgaudin** [comment]
> @rgaudin Does it fix #141

No

**Comment by rgaudin** [comment]
Updated to use 7.2.2 instead

**Review by kelson42 — APPROVED** [review]

**Review by mgautierfr — APPROVED** [review]

**Pull request merged**
